### ISSUE TITLE
Request - Walking on phone

### ISSUE THREAD

**Issue by odivalq** [criffin/vr-natural-movement#1]
Hello, I would like to request that you make jogging and walking being tracked using the phones accelerometer - similar to the hand moviment feature. This way people with DIY headsets can also benefit of walking inside games. Could this be possible?

**Comment by filippkeks** [comment]
It is absolutely possible. Probably we will implement this when we have time. Meanwhile you may help us with the implementation and send a pull request ;)

**Comment by odivalq** [comment]
Unfortunately I'm know nothing of coding beyond the bare minimum C# for Unity 3D development and so I'll have to pass. But if this ever become a feature I would apreciate immensely. =x

**Comment by dfoleyAFG** [comment]
This is possible and I have implemented it in multiple games. Its fairly simple accelerometer range to get an accurate "step". I could help implement.

**Comment by pottedmeat7** [comment]
This is possible and I have implemented it in multiple games. Its fairly simple accelerometer range to get an accurate "step". I could help implement.

**Comment by odivalq** [comment]
Yeah, I also ended up learning how to do so. It's basically a podometer algorithm, will fish the code as soon as possible, but I would very much like to see your implementation as well pottedmeat7, mine was kinda crude =x

**Comment by pottedmeat7** [comment]
Hi was just looking more into some of the descriptions of methods here. Not sure if you guys have used this in games much... But I saw that your current method for "step detection" is to track where the "head" is and its vertical position. This seems like an inefficient way to detect a step, and using the phones accelerometer works very well. Wondering if you have tried it and can merge these branches.

**Comment by MuhammadBinSaif** [comment]
Hand movement is not working when i build app for andriod and run app there will be no response from andriod app nor from pc app.

**Comment by MuhammadBinSaif** [comment]
kidly do reply soon
